docs/docs/snapshots.md

```diff
@@ -10,13 +10,13 @@ You can download a snapshot of the Juno database to reduce the network syncing t
 
 | Version      | Size       | Block      | Download Link                                                                                         |
 | ------------ | ---------- | ---------- | ----------------------------------------------------------------------------------------------------- |
-| **>=v0.9.2** | **172 GB** | **654464** | [**juno_mainnet.tar**](https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.11.9_654464.tar) |
+| **>=v0.9.2** | **172 GB** | **654464** | [**juno_mainnet.tar**](https://juno-snapshots.nethermind.dev/mainnet/juno_mainnet_v0.11.9_654881.tar) |
 
 ## Sepolia
 
 | Version      | Size     | Block     | Download Link                                                                                        |
 | ------------ | -------- | --------- | ---------------------------------------------------------------------------------------------------- |
-| **>=v0.9.2** | **5 GB** | **66477** | [**juno_sepolia.tar**](https://juno-snapshots.nethermind.dev/sepolia/juno_sepolia_v0.11.7_66477.tar) |
+| **>=v0.9.2** | **5 GB** | **654881** | [**juno_sepolia.tar**](https://juno-snapshots.nethermind.dev/sepolia/juno_sepolia_v0.11.7_66477.tar) |
```

Comment:
if snapshot link is the same why do we need to change it here?

Reply:
Yep, my bad - fixed